specification/http/1.0/openapi.json

```diff
@@ -0,0 +1,456 @@
+    }
+  },
+  "paths": {
+    "/health": {
+      "get": {
+        "summary": "Check health",
+        "description": "Checks the health status of the server.",
+        "responses": {
+          "200": {
+            "description": "Server is healthy."
+          }
+        }
+      }
+    },
+    "/tools": {
+      "get": {
+        "summary": "List tools",
+        "description": "Returns a list of tool definitions.",
+        "security": [
+          {
+            "bearerAuth": []
+          },
+          {}
+        ],
+        "responses": {
+          "200": {
+            "description": "A list of available tools.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "properties": {
+                    "$schema": {
+                      "type": "string",
+                      "format": "uri"
+                    },
+                    "tools": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/components/schemas/ToolDefinition"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/execute": {
+      "post": {
+        "summary": "Execute tool",
+        "description": "Executes a tool with the given parameters.",
+        "security": [
+          {
+            "bearerAuth": []
+          },
+          {}
+        ],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "properties": {
+                  "$schema": {
+                    "type": "string",
+                    "format": "uri"
+                  },
+                  "request": {
+                    "$ref": "#/components/schemas/ExecuteToolRequest"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Execution response from tool.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "properties": {
+                    "$schema": {
+                      "type": "string",
+                      "format": "uri"
+                    },
+                    "response": {
+                      "$ref": "#/components/schemas/ExecuteToolResponse"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "components": {
+      "securitySchemes": {
+        "bearerAuth": {
+          "type": "http",
+          "scheme": "bearer",
+          "bearerFormat": "JWT"
+        }
+      },
+      "schemas": {
+        "ToolId": {
+          "type": "string",
+          "description": "A tool's unique identifier in the format 'Toolkit.Tool[@version]', where @version is optional.",
```

Comment:
* Does `latest` have latest semantics?
* If not, what is the expected behavior when a version number isn't provided?

Reply:
1. Yes
2. Also `latest`

I made a TODO note to flesh this out in more detail in the RFC itself.